src/main/java/com/Employes/Employedetail/service/employeeservice.java

```diff
@@ -0,0 +1,328 @@
+package com.Employes.Employedetail.service;
+
+import com.Employes.Employedetail.Entity.Designation;
+import com.Employes.Employedetail.Entity.Employee;
+import com.Employes.Employedetail.controller.postRequest;
+import com.Employes.Employedetail.controller.putRequest;
+import com.Employes.Employedetail.repository.DesignationRepo;
+import com.Employes.Employedetail.repository.Employerepo;
+import com.Employes.Employedetail.util.MessageUtil;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+//SERVICE CLASS WHICH CARRY OUT THE REST API SERVICES
+@Service
+public class employeeservice extends employeevalidate{
+@Autowired
+    Employerepo repo;
+@Autowired
+    DesignationRepo depo;
+@Autowired
+    MessageUtil messageUtil;
```

Comment:
Java access modifier are missing above object. Default access modifier is package.